wdl/ExpansionHunter.wdl

```diff
@@ -142,3 +165,145 @@ task RunExpansionHunter {
         maxRetries: runtime_attr.max_retries
     }
 }
+
+task SplitVariantCatalog {
+    input {
+        File variant_catalog
+        Int batch_size
+        String? split_variant_catalog_docker
+        String? output_prefix
```

Comment:
I think it's better to make these required. it's easier to keep track of what dockers are being used across the whole pipeline by just being able to reference `/inputs/dockers.json` rather than having to comb through all the WDLs. I'd also argue for a required prefix since, as a general practice, it tends to be easier to keep track of output files (i.e. when we have to copy them to a single directory for debugging) when the output names are unique.
```suggestion
        String? python_docker
        String? output_prefix
```

Reply:
Good suggestions, thank you! changed accordingly.